processor/elasticprocessor/internal/hostmetrics/metric.go

```diff
@@ -47,6 +47,23 @@ func addMetrics(ms pmetric.MetricSlice, dataset string, metrics ...metric) {
 			dp.SetStartTimestamp(metric.startTimestamp)
 		}
 
+		process_ppid, _ := rm.Attributes().Get("process.parent_pid")
```

Comment:
shouldn't these be added in the process specific code?

Reply:
additionally, what's the reason for copying these resource attributes to data point attributes?

Reply:
ah right, thanks for the reminder.